src/daemon/main.go

```diff
@@ -363,8 +363,18 @@ func main() {
 	}
 
 	log.Infof("%s", banner(cfg))
+	// Used to not log out the proxy, which has potentially secret credentials
+	var proxy bool = false
 	for i := range os.Args {
-		log.Debugf("ARGV[%d]: %s", i, os.Args[i])
+		if proxy {
+			log.Debugf("ARGV[%d]: **REDACTED**", i)
+			proxy = false
+		} else {
+			log.Debugf("ARGV[%d]: %s", i, os.Args[i])
```

Comment:
Should this `log.Debugf()` be here since the proxy check happens after that line?

Reply:
Argv[] splits the parameter name and value into 2 separate entries. So the proxy check is looking for "--proxy" and then the next loop iteration will be the actual proxy value.